src/system-health/health_checker/hardware_checker.py

```diff
@@ -283,3 +290,58 @@ def _ignore_check(cls, ignore_set, category, object_name, check_point):
         elif '{}.{}'.format(object_name, check_point) in ignore_set:
             return True
         return False
+
+    def publish_events(self, sensors, event_name):
+        params = swsscommon.FieldValueMap()
+        events_handle = swsscommon.events_init_publisher(EVENTS_PUBLISHER_SOURCE)
+        for sensor in sensors:
+            params[event_name] = sensor
+            swsscommon.event_publish(events_handle, EVENTS_PUBLISHER_TAG, params)
+        swsscommon.events_deinit_publisher(events_handle)
+
+
+    def _check_liquid_cooling_status(self, config):
+        """
+        Check liquid cooling status including:
+            1. Check all leakage sensors are in good state
+        :param config: Health checker configuration
+        :return:
+        """
+        if config.user_defined_checkers and 'liquid_cooling' not in config.user_defined_checkers:
```

Comment:
Maybe we should do it like this: 
1. Add a new config section to configeration, like "config.include_devices"
2. Change the code here to:
```python
if not config.include_devices or 'liquid_cooling' not in config.include_devices:
```